dspace/config/registries/uclouvain-additions-types.xml

```diff
@@ -494,6 +505,31 @@
         <qualifier>issn</qualifier>
         <scope_note>To specify a potential issn identifier for the serial</scope_note>
     </dc-type>
+    <!-- DISSERTATION SCHEMA -->
+    <dc-schema>
+        <name>dissertation</name>
+        <namespace>https://uclouvain.be/ontology/dissertation</namespace>
+    </dc-schema>
+    <dc-type>
+        <schema>dissertation</schema>
+        <element>institution</element>
+        <scope_note>The main institution of the dissertation</scope_note>
+    </dc-type>
+    <dc-type>
+        <schema>dissertation</schema>
+        <element>degree</element>
```

Comment:
add "name" qualifier and create a new field for "dissertation.degree.code" (could be use to improve data exposure using OAI-PMH)